ci-operator/config/openshift/release/openshift-release-master__nightly-4.16.yaml

```diff
@@ -84,15 +84,69 @@ releases:
       stream: nightly
       version: "4.16"
   oldest-supported-z-rollback:
-    release:
-      channel: candidate
-      version: 4.16.0-ec.0
+    candidate:
+      product: ocp
+      relative: 1
+      stream: nightly
+      version: "4.16"
 resources:
   '*':
     requests:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: aws-ipi-byo-route53-upgrade-rollback-oldest-supported
+  cron: 18 18 7,14,23,30 * *
+  steps:
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:oldest-supported-z-rollback
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    test:
+    - as: set-upgrade-releases
+      commands: echo "${RELEASE_IMAGE_LATEST},${RELEASE_IMAGE_Z_ROLLBACK}" > ${SHARED_DIR}/upgrade-edge
+      dependencies:
+      - env: RELEASE_IMAGE_Z_ROLLBACK
+        name: release:oldest-supported-z-rollback
+      - env: RELEASE_IMAGE_LATEST
+        name: release:latest
+      from: cli
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    - ref: cucushift-chainupgrade-toimage
```

Comment:
Just happen to find there is a new sub-command `rollback` for 4.16 z-rollback. It looks like merged for some time in https://github.com/openshift/oc/pull/1642. 

```
# ./oc adm upgrade rollback --help
Rollback the cluster to the previous release.

 Only patch version rollbacks within the same z stream, e.g. 4.y.newer to 4.y.older, are supported. Minor version
rollbacks from 4.newer to 4.older are not supported.  Updates to releases other than the most-recent previous release
that the cluster was attempting are not supported.  Rolling back re-exposes the cluster to all the bugs which had been
fixed from 4.y.older to 4.y.newer.  In most cases, you probably want to understand what's having trouble and roll
forward with fixes.

Usage:
  oc adm upgrade rollback [flags] [options]

Use "oc options" for a list of global command-line options (applies to all commands).
```

hmm, so shall we get a dedicated step for the new cmd to integrate it into ci for the rollback test? at least for 4.16.

Reply:
Good finding. `oc adm upgrade rollback` allows rollback to the most recent previous version which should be less than the current version. In the test, we are testing 4.y.0-> 4.y.0-0.nightly-> 4.y.0, where 4.y.0 is greater than 4.y.0-0.nightly. So `oc adm upgrade rollback` will not work here. We can use  the command in stable to stable test.